src/magic_enchantment.cpp

```diff
@@ -286,9 +283,18 @@ void enchantment::load( const JsonObject &jo, const std::string &,
     }
 
     active_conditions.first = io::string_to_enum<has>( jo.get_string( "has", "HELD" ) );
-    active_conditions.second = io::string_to_enum<condition>( jo.get_string( "condition",
-                               "ALWAYS" ) );
-
+    if( jo.has_member( "condition" ) ) {
+        cata::optional<enchantment::condition> con = io::string_to_enum_optional<condition>
```

Comment:
doing it this way will break copy-from. use something like `optional` instead here.

Reply:
Ok I've tried to address this but I don't know copy-from well enough to know if its enough.  Also its a hard situation because you can have a string that is either an enum or a condition.